.github/workflows/ci.yml

```diff
@@ -27,10 +24,20 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ matrix.rust }}
-          target: ${{ matrix.TARGET }}
+          target: thumbv6m-none-eabi thumbv7m-none-eabi thumbv7em-none-eabi thumbv7em-none-eabihf thumbv8m.main-none-eabi
```

Comment:
I don't think this action supports lists https://github.com/actions-rs/toolchain/pull/62

Reply:
It might be easiest to just invoke rustup manually, it *can* take a list of targets to install

Reply:
Yes indeed. I'm alreay working on it.